pkg/selectors/kernel.go

```diff
@@ -521,7 +521,7 @@ func writeMatchValuesRange(k *KernelSelectorState, values []string, ty uint32) e
 		}
 
 		switch ty {
-		case gt.GenericIntType, gt.GenericS32Type, gt.GenericSizeType, gt.GenericU32Type:
+		case gt.GenericIntType, gt.GenericS32Type, gt.GenericSizeType, gt.GenericU32Type, gt.GenericU16Type, gt.GenericU8Type, gt.GenericS16Type, gt.GenericS8Type:
```

Comment:
suggestion: Would it make sense to extend the TestUInt16Tracepoint test in this commit to also add a filter to test this? (I'm assuming this adds support for filters).